CONTRIBUTING.md

```diff
@@ -25,8 +25,40 @@ you have to run the following command:
 git config --local blame.ignoreRevsFile .git-blame-ignore-revs
 ```
 
-## Change the formatting
+### Change the formatting
 
 If there is a consensus between the developpers, the formatting can be
 changed. Do not hesitate to make suggestions. In that case, do not
 forget to edit the `.git-blame-ignore-revs` file.
+
+## Writing tests
+
+Tests was originally written with `bash` scripts. We prefer now to use
+[Tezt](https://tezos.gitlab.io/developer/tezt.html?highlight=tezt)
+which is a test framework originally designed for
+[Tezos](https://tezos.gitlab.io).
+
+We invite and recommand developers to write new integration tests
+using this framework.
+
+### Useful tips with Tezt
+
+- You can run `Tezt` tests with `dune exec tests/main.exe`
+
+- You can list all the tests with `dune exec tests/main.exe -- --list`
+
+- You can filter tests by tags. For example: `dune exec tests/main.exe -- check`.
+   Tags can be negated or combined (see `dune exec tests/main.exe -- --help`)
+
+- You can execute a single test using its name. For example: `dune
+  exec tests/main.exe -- --title "check 'recursive.dk' succeeds"`
+
+- Reset regression tests with `dune exec tests/main.exe -- --reset-regressions`.
```

Comment:
What does reset mean? Are there regression and other-than-regression tests?

Reply:
It means when the regression output changed, you can reset the regression output with this option. I will clarify this.

Reply:
Fix in ed6d39f5